pyopendds/dev/include/pyopendds/basictype.hpp

```diff
@@ -29,6 +29,7 @@ class BooleanType {
         } else {
             py = Py_True;
         }
+        Py_INCREF(py);
```

Comment:
the REF is never released ?

Reply:
https://docs.python.org/3/c-api/bool.html

maybe use `PyBool_FromLong(cpp)` and remove the `Py_INCREF()`